c65/c65.c

```diff
@@ -134,7 +184,7 @@ int save_memory(const char* romfile, uint16_t start, uint16_t end) {
 void show_cpu() {
   printf(
       "c65: PC=%04x A=%02x X=%02x Y=%02x S=%02x FLAGS=<N%d V%d B%d D%d I%d Z%d "
-      "C%d> ticks=%lu\n",
+      "C%d> ticks=%llu\n",
```

Comment:
I see only two %llu errors left (you got most of them already, and your fix compiles cleanly in the places you've already changed).  I'll flag them for you as you won't see the error on your system.  This one is in `void show_cpu()` on line 187 of c65.c.

Reply:
ah, %llu instead of %llx 🤦